index.js

```diff
@@ -398,11 +398,14 @@ function parseBitmap (data, path, cb) {
             }
 
         case Jimp.MIME_TIFF:
-            var tiff = (UTIF.decode(data)[0]);
+            var ifds = UTIF.decode(data)
+            var page = ifds[0];
+            UTIF.decodeImages(data, ifds);
```

Comment:
Is this line actually doing anything?

Reply:
Which line ? `var ifds = UTIF.decode(data)` ?

Reply:
I believe it modifies `ifds` in place.
I need to re-check this